benchmarks/gravity_prem/doc/gravity_prem.tex

```diff
@@ -0,0 +1,295 @@
+
+\begin{lstlisting}
+subsection Postprocess
+  set List of postprocessors = gravity calculation
+  subsection Gravity calculation
+    set Sampling scheme             = list of points 
+    set List of radius              = 6596000
+    set List of longitude           = 38.7578, 174.7633, -157.8583,   85.324, -77.0428,   2.3522, -21.9426, -122.4194, 139.6917,  16.3738
+    set List of latitude            =  8.9806, -36.8485,   21.3069,  27.7172, -12.0464,  48.8566,  64.1466,   37.7749,  35.6895,  48.2082 
+  end
+end
+\end{lstlisting}
+
+Results are shown in Fig.~\ref{fig:gravitycities}. Measurements agree well with the expected analytical value but the presence of under- and overshoot is the unavoidable consequence of the type of mesh used and quadrature order.
+  
+\begin{table}
+\centering
+\begin{tabular}{llll}
+\hline
+Index & City & Latitute & Longitude \\
+\hline\hline
+ 0 & Addis Ababa &8.9806\si{\degree}N & 38.7578\si{\degree}E \\
+ 1 & Auckland & 36.8485\si{\degree}S & 174.7633\si{\degree}E \\
+ 2 & Honolulu &21.3069\si{\degree}N & 157.8583\si{\degree}W \\
+ 3 & Kathmandu &27.7172\si{\degree}N & 85.3240\si{\degree}E \\
+ 4 & Lima &12.0464\si{\degree}S & 77.0428\si{\degree}W \\
+ 5 & Paris & 48.8566\si{\degree}N & 2.3522\si{\degree}E \\
+ 6 & Reykjavik & 64.1466\si{\degree}N & 21.9426\si{\degree}W \\
+ 7 & San Francisco & 37.7749\si{\degree}N & 122.4194\si{\degree}W \\ 
+ 8 & Tokyo & 35.6895\si{\degree}N & 139.6917\si{\degree}E \\
+ 9 & Vienna &48.2082\si{\degree}N & 16.3738\si{\degree}E \\
+\hline
+\end{tabular}
+\caption{List of the 10 implemented cities and their latitude/longitude coordinates.}
+\label{tab:cities}
+\end{table}
+
+\begin{figure}[h!]
+\centering
+\includegraphics[width=0.48\linewidth]{../../benchmarks/gravity_prem/doc/profile_gravity_const_g.pdf}
+\includegraphics[width=0.48\linewidth]{../../benchmarks/gravity_prem/doc/profile_gravity_const_U.pdf}
+\caption{Gravity acceleration $|{\mathbf g}|$ and potential $U$ along a radial profile for a hollow sphere of constant density of 3000 \si{\kilogram\per\cubic\metre}. The grey area symbolises the hollow sphere interior where density is non zero. The analytical solution is also shown in black.}
+\label{fig:gravityline}
+\end{figure}
+
+ \begin{figure}[h!]
+\centering
+\includegraphics[width=0.48\linewidth]{../../benchmarks/gravity_prem/doc/cities_gravity_const_g.pdf}
+\includegraphics[width=0.48\linewidth]{../../benchmarks/gravity_prem/doc/cities_gravity_const_U.pdf}
+\caption{Gravity acceleration $|{\mathbf g}|$ and potential $U$ at GOCE height  (225 \si{km}) above 10 cities query points for a hollow sphere of constant density of 3000 \si{\kilogram\per\cubic\metre} counting 61440 elements. We varied the increased quadrature point degree from 2 to 4 to illustrate the rapid convergence of the gravity and potential toward the analytical solution shown in black.}
```

Comment:
In this figure it looks like the optimal quadrature degree is nq+3 (I guess that means the degree of the finite element plus 3?). It is slightly surprising that nq+4 is worse, I would expect it to get better with increasing quadrature degree until at some point it does not make a difference. Can you check that the captions and the plotted lines are consistent?

Reply:
Well spotted! I missed that and for this case, there has been a swap between nq+3 and nq+4. That is because I ran the models individually and an error occurred. It is indeed expected at a specific height, increasing the quadrature degree results in a convergence. as shown in this figure: 




However, I was not initially surprised because I did see cases with other experiments that I am performing now, where the nq+4 is worse than nq+3. Those new experiments are ran in a shellscript using varied GMR from 0 to 4 and increase in nq+ from 0 to 7. Then the gnuplot is also from a loop. 
Here are a series of figures showing gravity acceleration difference between gravity measured and the gravity theoretic along a radial profile for a sphere with PREM densities. You can see that increasing nq+X does not predict a smooth convergence. 

![B2_profile_g_gmr_nq0_diff](https://user-images.githubusercontent.com/32137895/97480714-1712fe00-1954-11eb-80f4-eb5400032ee5.png)
![B2_profile_g_gmr_nq7_diff](https://user-images.githubusercontent.com/32137895/97480718-17ab9480-1954-11eb-9992-376dd73c63be.png)
![B2_profile_g_nq_gmr1_diff](https://user-images.githubusercontent.com/32137895/97480719-17ab9480-1954-11eb-8ec4-8a37bb956023.png)
![B2_profile_g_nq_gmr4_diff](https://user-images.githubusercontent.com/32137895/97480720-18442b00-1954-11eb-8e23-a5c2088601a9.png)